tools/lib/test_server.py

```diff
@@ -62,6 +62,7 @@ def test_server_running(force=False, external_host='testserver',
 
     set_up_django(external_host)
 
+    use_db = True
```

Comment:
This sort of change should definitely be its own commit, since it's a behavior change.

It's also the wrong fix; we should be doing it in the test_help_documentation script.  I applied the fix there.

Reply:
Whoops, sorry, that was my local hotfix, didn't intend it to go in here.